commit c813c0305ae36d6430245354fe4aef3ccd7a59e6
Author: Phil Elwell <phil@raspberrypi.org>
Date:   Tue May 16 14:39:49 2017 +0100

    mmc: Change downstream MMC driver CONFIG option
    
    The upstream SDHOST driver has now claimed CONFIG_MMC_BCM2835, which
    clashes with the downstream MMC driver. Rename the downstream option to
    CONFIG_MMC_BCM2835_MMC.
    
    Signed-off-by: Phil Elwell <phil@raspberrypi.org>

diff --git a/drivers/mmc/host/Kconfig b/drivers/mmc/host/Kconfig
index 6300af1eff1a..e577d20963bc 100644
--- a/drivers/mmc/host/Kconfig
+++ b/drivers/mmc/host/Kconfig
@@ -4,7 +4,7 @@
 
 comment "MMC/SD/SDIO Host Controller Drivers"
 
-config MMC_BCM2835
+config MMC_BCM2835_MMC
 	tristate "MMC support on BCM2835"
 	depends on MACH_BCM2708 || MACH_BCM2709 || ARCH_BCM2835
 	help
@@ -16,7 +16,7 @@ config MMC_BCM2835
 
 config MMC_BCM2835_DMA
 	bool "DMA support on BCM2835 Arasan controller"
-	depends on MMC_BCM2835
+	depends on MMC_BCM2835_MMC
 	help
 	  Enable DMA support on the Arasan SDHCI controller in Broadcom 2708
 	  based chips.
diff --git a/drivers/mmc/host/Makefile b/drivers/mmc/host/Makefile
index f4b8951af214..d352fabf6b61 100644
--- a/drivers/mmc/host/Makefile
+++ b/drivers/mmc/host/Makefile
@@ -18,7 +18,7 @@ obj-$(CONFIG_MMC_SDHCI_S3C)	+= sdhci-s3c.o
 obj-$(CONFIG_MMC_SDHCI_SIRF)   	+= sdhci-sirf.o
 obj-$(CONFIG_MMC_SDHCI_F_SDH30)	+= sdhci_f_sdh30.o
 obj-$(CONFIG_MMC_SDHCI_SPEAR)	+= sdhci-spear.o
-obj-$(CONFIG_MMC_BCM2835)	+= bcm2835-mmc.o
+obj-$(CONFIG_MMC_BCM2835_MMC)	+= bcm2835-mmc.o
 obj-$(CONFIG_MMC_BCM2835_SDHOST)	+= bcm2835-sdhost.o
 obj-$(CONFIG_MMC_WBSD)		+= wbsd.o
 obj-$(CONFIG_MMC_AU1X)		+= au1xmmc.o
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1127_c813c0305ae36d6430245354fe4aef3ccd7a59e6.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1127_c813c0305ae36d6430245354fe4aef3ccd7a59e6.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
